apps/charterafrica/src/payload/collections/Tag.js

```diff
@@ -0,0 +1,21 @@
+const Tag = {
+  slug: "tag",
+  admin: {
+    useAsTitle: "tagName",
+  },
+  fields: [
+    {
+      name: "tagName",
```

Comment:
```suggestion
      name: "name",
```

We know it's a tag already.